kubernetes/resources_api/overlays/staging/deployment.yaml

```diff
@@ -0,0 +1,13 @@
+
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: resources-api
+spec:
+  template:
+    spec:
+      containers:
+      - name: app
+        env:
+        - name: FLASK_ENV
```

Comment:
Question only: Is this a carryover from `RAILS_ENV`? I don't think this is a normal Flask convention.

Reply:
`FLASK_ENV` defaults to production. 

when calling `flask run` it will look for `FLASK_ENV` and if it's equal to `development` has the same behavior as `FLASK_DEBUG` 

This isn't necessarily needed here. And I'd think it might be worthwhile to take it out unless we have some functionality that works in production but not staging.

Reply:
http://flask.pocoo.org/docs/1.0/config/#environment-and-debug-features

Reply:
I thought that it might be useful for logging purposes. If something gets alerted on (idk if we use something like bugsnag, maybe prometheus has alerts??) then it would be useful to know that it happened in staging rather than production. If our testing/staging environment doesn't have anything like that, then it really doesn't matter. As far as the API is concerned, "staging" and "production" are both non-development, so the behavior ought to be the same

Reply:
I'm ok with removing it if one or both of you believe it's overkill

Reply:
If you end up using sentry. Then you can configure sentry to alert based on the environment and use the above FLASK_ENV as the appropriate environment.